### PR TITLE
View/ui improvement

### PR DESCRIPTION
**Pull request by aburgosu**

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/GeneralSearchPanel.java [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+    private JDateChooser FieldDateAccessFrom;
+    private JDateChooser FieldDateAccessTo;
+    private JTextField textFieldSizeFrom;
+    private JTextField textFieldSizeTo;
+    private JButton btnSearch;
+
+    /**
+     * Initializes a SearchTabs instance for the search options
+     */
+    public GeneralSearchPanel(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/GeneralSearchPanel.java]: put space after )

File: src/main/java/com/fundation/logic/view/GeneralSearchPanel.java [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+        gbc_btnSearch.insets = new Insets(0, 0, 5, 5);
+        gbc_btnSearch.gridx = 1;
+        gbc_btnSearch.gridy = 13;
+        add(btnSearch, gbc_btnSearch);
+    }
+
+    /**
+     * This method return the Search Button
+     */
+    public JButton getSearchButton(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/GeneralSearchPanel.java]: put space after ")"

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 3]
```diff
@@ -52,6 +53,7 @@ public void initComponent() {
 
         searchTabs = new SearchTabs();
         mainSplitPanel.setLeftComponent(searchTabs);
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java:56]: delete empty line

File: src/main/java/com/fundation/logic/view/GeneralSearchPanel.java (hunk 1 above)
Comment (aburgosu) [reply src/main/java/com/fundation/logic/view/GeneralSearchPanel.java]: Done

File: src/main/java/com/fundation/logic/view/GeneralSearchPanel.java (hunk 2 above)
Comment (aburgosu) [reply src/main/java/com/fundation/logic/view/GeneralSearchPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 3 above)
Comment (aburgosu) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java:56]: done

**Pull request merged**